mappings/net/minecraft/item/Item.mapping

```diff
@@ -1,149 +1,182 @@
 CLASS bbw net/minecraft/item/Item
 	CLASS bbw$a Settings
-		FIELD a fullStackSize I
-		FIELD b durability I
+		FIELD a maxCount I
+		FIELD b maxDamage I
 		FIELD c recipeRemainder Lbbw;
-		FIELD d itemGroup Lbas;
+		FIELD d group Lbas;
```

Comment:
Is just `group` a bit vague?

Reply:
It's the group (`ItemGroup`) bound to the item. The getter is also `getGroup()`, so you would call as `item.getGroup()` returning an `ItemGroup`.

Reply:
Ah yeah, thats a fair point.